src/index.js

```diff
@@ -1,21 +1,54 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-cycle */
 /* eslint-disable no-plusplus */
 /* eslint-disable no-unused-vars */
 import _ from 'lodash';
 import './style.css';
+import { checked, textDecorationChecked, changeState } from './checked.js';
 
 const ul = document.querySelector('.task-list');
 
-const activities = [{ description: 'example 1', completed: false, index: 0 }, { description: 'example 2', completed: false, index: 1 }, { description: 'example 3', completed: false, index: 2 }];
```

Comment:
- [ ] The array of object shouldn't be removed, it should be there and add the function to the array see the number 4 in the requirements 👇 it stays the array should be stored, and when I check the check box It should update from false to true see number 1 and 2
![,,,,,,,llll](https://user-images.githubusercontent.com/61562626/130264347-4778e1ef-aca9-4769-8a62-2bccf23fb8b8.jpeg)